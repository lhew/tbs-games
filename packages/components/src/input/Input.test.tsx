import React from 'react';
import TestRenderer from 'react-test-renderer'; // ES6
import Input from '.';

test('Input scenarios', () => {
  const defaultInput = TestRenderer.create(<Input type="text" value="foo" />);
  expect(defaultInput).toMatchSnapshot();
});
