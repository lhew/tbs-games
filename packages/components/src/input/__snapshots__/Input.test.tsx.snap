// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input scenarios 1`] = `
<input
  className="sc-fzozJi fbOHBz"
  type="text"
  value="foo"
/>
`;
